apps/src/applab/designMode.js

```diff
@@ -394,8 +394,11 @@ designMode.parseFromLevelHtml = function(rootEl, allowDragging) {
   }
 
   children.each(function () {
-    elementLibrary.onDeserialize($(this)[0]);
+    elementLibrary.onDeserialize($(this)[0], designMode.onPropertyChange.bind(this));
   });
+  children.children().each(function() {
+    elementLibrary.onDeserialize($(this)[0], designMode.onPropertyChange.bind(this));
+  })
```

Comment:
I went to do this and the original code still seems simpler than anything else I came up with. Moving the logic into elementLibrary.onDeserialize requires doing one of: extra work (checking 3rd level children), extra depth checks, or duplicate logic to explicitly only check the second level. If we ever have elements that are 3 levels deep then this sounds like the way to go, though.